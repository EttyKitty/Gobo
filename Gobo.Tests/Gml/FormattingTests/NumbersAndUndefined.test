x = 000
y = 001230
z = 000.4560
i = .789
j = 0b010101
k = $abcd
a = 0xabcd
b = #2c8edd
c = NaN
d = infinity

foo = 21474836470000000000
bar = .1234567890123456789024

var _integer = 100_000_000; // same as 100000000
var _float   = 3_141.59; // same as 3141.59
var _hexadec = 0xDEAD_BEEF; // same as 0xDEADBEEF
var _binary  = 0b01101000_01101001; // same as 0b0110100001101001

call(,,)

call( , /*comment*/ , )

call(, // surprise! inner comment
,)