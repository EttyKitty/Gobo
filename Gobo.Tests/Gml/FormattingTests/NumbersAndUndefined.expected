x = 0;
y = 1230;
z = 0.4560;
i = 0.789;
j = 0b010101;
k = $abcd;
a = 0xabcd;
b = #2c8edd;
c = NaN;
d = infinity;

foo = 21474836470000000000;
bar = 0.1234567890123456789024;
baz = 123.0;

var _integer = 100_000_000; // same as 100000000
var _float = 3_141.59; // same as 3141.59
var _hexadec = 0xDEAD_BEEF; // same as 0xDEADBEEF
var _binary = 0b01101000_01101001; // same as 0b0110100001101001

call(, , );

call(, /*comment*/ , );

call(
	, // surprise! inner comment
	,

);
